iron-request.html

```diff
@@ -425,7 +425,9 @@
     _wwwFormUrlEncodePiece: function(str) {
       // Spec says to normalize newlines to \r\n and replace %20 spaces with +.
       // jQuery does this as well, so this is likely to be widely compatible.
-      if (str === null) {
+      try {
+        str.toString
```

Comment:
The issue this solves in particular is that `null` and `undefined` throw an exception when you do

```javascript
var a = null;
var b = undefined;
a.toString; // throws exception
b.toString; // throws exception
```
Perhaps the user would want to submit `NaN` hence why I have this implementation.